overseerr-api.yml

```diff
@@ -186,6 +187,9 @@ components:
         defaultPermissions:
           type: number
           example: 32
+        enableForwardAuth:
+          type: boolean
+          example: true
```

Comment:
You wrote the setting here, but it is not available anywhere.

You should add this as a general setting, by adding a new properties in the `MainSettings` type in `server/lib/settings.ts` and adding a new input in the settings page in `src/components/Settings/SettingsMain/index.tsx`. You should also check that this setting is enabled before allowing login with forwardAuth.

Or did you intend to not make a setting and set it enabled by default?